libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -3,6 +3,19 @@
 
 $input-box-transitions: border-color 150ms, box-shadow 150ms, color 150ms;
 
+:root {
+  --sky-background-color-input-box-group: transparent;
+  --sky-background-color-input-box-group-focused: transparent;
+}
+
+.sky-theme-modern {
+  --sky-background-color-input-box-group-focused: #{$sky-color-white};
+
+  &.sky-theme-mode-dark {
+    --sky-background-color-input-box-group-focused: #{$sky-color-black};
```

Comment:
Ok, added the scope. With that scope, it can't reference the variable in the ag-grid stylesheet, so I switched ag-grid to use the background color property.